lib/loader/loader.js

```diff
@@ -3,15 +3,37 @@ var loader, define, requireModule, require, requirejs;
 (function(global) {
   'use strict';
 
+  var heimdall = global.heimdall;
+  var {
+    define: __define,
+    require: __require,
+    exports: __exports,
+    findModule: __findModule,
+    resolve: __resolve,
+    reify,
+    findDeps,
+    modules,
+    resolveRelative,
+    pendingQueueLength
+  } = heimdall.registerMonitor('loaderjs',
+  'define',
+  'require',
+  'reify',
+  'findDeps',
+  'modules',
+  'exports',
+  'resolve',
+  'resolveRelative',
+  'findModule',
+  'pendingQueueLength');
```

Comment:
@turbo87 if installed properly, dist will be there. If you are linking you may need to run the build script.